tests/manage/pv_services/test_create_pvc_with_worker_node_down.py

```diff
@@ -0,0 +1,98 @@
+import logging
+import random
+import pytest
+
+from ocs_ci.ocs.resources import pod
+
+from ocs_ci.ocs import node
+from tests import helpers
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import ManageTest, tier4, tier4a
+from tests.helpers import get_worker_nodes
+from tests.sanity_helpers import Sanity
+
+log = logging.getLogger(__name__)
+
+
+@tier4
+@tier4a
+class TestCreatePvcWithWorkerNodeDown(ManageTest):
+
+    @pytest.fixture(autouse=True)
+    def teardown(self, request, nodes):
+        """
+        Make sure all nodes are up again
+
+        """
+        def finalizer():
+            nodes.restart_nodes_teardown()
```

Comment:
Is that necessary for the teardown?
If we NEED to restart the nodes, does that mean that without restart, the cluster is in bad state?

Reply:
this fixture actually start VM which are in shutdown state I have added this in teardown fixture because if something bad happens in test execution teardown will bring the node back online

Reply:
@prsurve better to check which nodes are in NotReady state and restart those.
see reference in tests/manage/z_cluster/nodes/test_disk_failures.py:36

@RazTamir the reason for having this fixture is for situations in which the test will fail before all nodes were brought up

Reply:
Ok, good to have that safety check.
Can we perform this only 'if' (nodes are still down, etc)?
I want to ensure we will not miss bugs because of this uncontrolled restart

Reply:
oh now I realize that restart_nodes_teardown() does all these checks. So no change is needed

Reply:
Done

Reply:
will revet back the changes

Reply:
Done